css/interactive-guides/microprofile-config/mpconfig.css

```diff
@@ -14,6 +14,10 @@
   table-layout: fixed;
 }
 
+.propsTable, .propsTable td, .propsTable tr, .propsTable th:focus {
```

Comment:
Do you want to take off the outline only on th:focus and not the td or tr focus?